apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -103,22 +127,15 @@ class SectionProgress extends Component {
               scriptId={scriptId}
               onChange={this.onChangeScript}
             />
-            {lessons.length !== 0 &&
-              <LessonSelector
-                lessons={lessons}
-                onChange={this.onChangeLevel}
-              />
-            }
           </div>
           <div style={styles.viewToggleContainer}>
             <SectionProgressToggle />
           </div>
-          <div style={styles.viewCourseLink}>
-            {linkToOverview &&
-              <SmallChevronLink
-                link={linkToOverview}
-                linkText={i18n.viewCourse()}
-                isRtl={false}
+          <div style={styles.lessonSelectorContainer}>
+            {lessons.length !== 0 &&
```

Comment:
I think `lessons.length > 1` might be what we want, since if there's only one lesson it probably doesn't make sense to jump to anything? Or do we want to leave it for consistency?

Reply:
Oh sorry - realized this was the same as code before and isn't related to this PR.